vapoursynth/src/api.rs

```diff
@@ -265,14 +331,65 @@ impl API {
         }
     }
 
+    /// Installs a custom handler for the various error messages VapourSynth emits. The message
+    /// handler is currently global, i.e. per process, not per VSCore instance.
+    /// Returns a unique id for the handler.
+    ///
+    /// If no error handler is installed the messages are sent to the standard error stream.
+    ///
+    /// The callback arguments are the message type and the message itself. If the callback panics,
+    /// the process is aborted.
+    ///
+    /// This version does not allocate at the cost of accepting a function pointer rather than an
+    /// arbitrary closure. It can, however, be used with simple closures.
+    #[inline]
+    #[cfg(feature = "gte-vapoursynth-api-36")]
+    pub fn add_message_handler_trivial(self, callback: fn(MessageType, &CStr)) -> VSMessageHandlerId {
```

Comment:
By "newtype" I meant adding a
```rust
#[derive(Debug, Clone, Copy, Eq, PartialEq, Hash)]
pub struct MessageHandlerId(VSMessageHandlerId);
```
with private inner field into `vapoursynth::api`, so that it's only possible to obtain it from `add_message_handler` and pass only it (and not an arbitrary int) to `remove_message_handler`. Could you please do that?